lib/full-schema.js

```diff
@@ -101,6 +101,7 @@ export const marks = {
       'highlight-keys': {},
     },
     inclusive: false,
+    excludes: '',
```

Comment:
Please add a comment above this line explaining it.

Reply:
Done.